settlement/settlement.go

```diff
@@ -50,6 +50,8 @@ type Transaction struct {
 
 // AntifraudTransaction a transaction object that comes from antifraud
 type AntifraudTransaction struct {
+	// CreatedAt         time.Time       `json:"created_at"`
```

Comment:
Are these comments valid?

Reply:
no. looks like the last changes were clobbered.

Reply:
fixed in 7f0ed18